idl/makefile

```diff
@@ -1,5 +1,8 @@
 .PHONY: proto
 
+PROTOC_GEN_GO_BIN_PATH := $(if $(shell which protoc-gen-go),$(shell which protoc-gen-go),"${GOPATH}/bin/protoc-gen-go")
```

Comment:
nit: prefer isolated changes to their own PRs as a general practice

Reply:
Isolated the changes in a stacked PR